index.html

```diff
@@ -0,0 +1,204 @@
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- Start Headline -->
+    <section class="headline container mt-3">
+      <h1 class="d-block mb-2">
+        "Hello Future Programmers"
+      </h1>
+      <img src="./imgs/headline.png" alt="Creatve Programmers and UI UX Designers" class="col-lg-9 col-sm-10 col-12"/>
+      <p class="ms-md-5 ms-3">
+        if you want to be a professional web developer. 
+        So it's time to take our course and become a professional developer. 
+        We are a group of professional teachers.
+      </p>
+    </section>
+    <!-- Start Main Program -->
+    <section class="main-program container-fluid text-light text-center mb-4 py-5">
+      <h2 class="mb-5">
+        Main Program
+      </h2>
+      <div class="row">
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/design.png" class="card-img-top" alt="design">
+            <div class="card-body">
+              <h5 class="card-title">UI & UX design</h5>
+              <p class="card-text">
+                We will teach you how to design and become a professional designer.
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/html-css-js.png" class="card-img-top" alt="html-css-js">
+            <div class="card-body">
+              <h5 class="card-title">Web Development Basics</h5>
+              <p class="card-text">
+                you will learn web development basics which are HTML, CSS, and JS
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/react-redux.jpg" class="card-img-top" alt="react-redux">
+            <div class="card-body">
+              <h5 class="card-title">ReactJS & Redux</h5>
+              <p class="card-text">
+                We will teach you one of popular frameworks ReactJS and Redux to handle states
+                to make your work easer and professional
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/node-mongo.png" class="card-img-top" alt="Ruby & Ruby on rails">
+            <div class="card-body pt-0">
+              <h5 class="card-title">Ruby On Rails</h5>
+              <p class="card-text">
+                Ruby on Rails makes it much easier and more fun. 
+                It includes everything you need to build fantastic applications,
+              </p>
+            </div>
+          </div>
+        </div>
+        <a href="#" class="h4 mt-5">
+          SEE WHOLE PROGRAM
```

Comment:
- [ ] In the mobile version, the background color is wrong, please fix it and follow the original design:

Your design:

![imagen](https://user-images.githubusercontent.com/60631456/132547009-634fee02-b157-4f7b-a408-fa98895f06b5.png)

Original design:

![imagen](https://user-images.githubusercontent.com/60631456/132547050-e66232eb-c2b1-4fd4-a5d9-e3a17fd37263.png)

Reply:
Well addressed this!